cob_description/test/test_urdf.py

```diff
@@ -22,6 +22,7 @@
 
 import rospy
 import rosunit
+import roslib
```

Comment:
no need to import it.
The files from `roslib` will be used by catkin to set-up `ROS_PACKAGE_PATH` automatically.

Reply:
And you need to import rospy..